airflow/providers/apache/beam/operators/beam.py

```diff
@@ -606,22 +606,6 @@ def execute_sync(self, context: Context):
                 )
 
     async def execute_async(self, context: Context):
```

Comment:
Given that you've removed asyncio loop from the `execute_async` method, does it still need to be an async method?

Reply:
A good note I think not. I will check the code and refactor it if it is needed.

Reply:
@e-galan I have updated the code. Could you please check one more time?